pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -130,9 +138,12 @@ subsets:
       - getRegionTimeFromVisit
       - diaPipe
       - injectedMatch
+      - consolidateMatchDiaSrc
       - sampleSpatialMetrics
       - analyzeAssocDiaSrcCore
       - analyzeTrailedDiaSrcCore
+      - analyzeDiaFakesDetectorVisitCore
+      - analyzeDiaFakesVisitCore
```

Comment:
These need to be mirrored in all instrument-specific pipelines that need to redefine the subsets (in this case, just DECam). `ap_verify` won't work correctly if the `apPipe` subset is missing tasks, so it's important for the reviewer to watch out for stuff like this.

Reply:
Filed as #195.